gm4_liquid_tanks/data/liquid_tanks/functions/init.mcfunction

```diff
@@ -4,16 +4,19 @@ execute unless entity @p run say GM4: Installing Liquid Tanks...
 
 #declare and initialise scoreboards and settings
 scoreboard players set update_happened gm4_upCheck 1
-scoreboard players set liquid_tanks gm4_modules 1
+scoreboard players set liquid_tanks gm4_modules 2
```

Comment:
Don't manually increment this score

Reply:
Oh, I did not know the site does this. This is an issue in #237.